website/versioned_docs/version-v0.13.0/getting-started.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Getting started
+id: version-v0.13.0-getting-started
+original_id: getting-started
+---
+
+This page shows how to get started with verifying or issuing IRMA attributes, using the following components:
+
+ * [`irma server`](irma-server.md), a server that verifies or issues IRMA attributes to [IRMA apps](irma-app.md),
+ * [`irma-frontend`](irma-frontend.md), a JavaScript library for drawing the IRMA QR in your website, and handling IRMA session with the `irma server`.
+
+You should have the [IRMA app](irma-app.md) installed ([Android](https://play.google.com/store/apps/details?id=org.irmacard.cardemu), [iOS](https://itunes.apple.com/nl/app/irma-authentication/id1294092994)). If you want to compile from source instead of using prebuilt binaries, you should additionally have [Git](https://git-scm.com/), [Go](https://golang.org/doc/install), and [npm](https://docs.npmjs.com/cli/npm) installed.
```

Comment:
Shouldn't it be better to start naming this `Yivi app` by now? Maybe make a reference like `Yivi app (formerly known as IRMA)`

Reply:
Check #53